packages/ts/react-crud/src/autoform.tsx

```diff
@@ -200,7 +205,7 @@ export function ExperimentalAutoForm<M extends AbstractModel>({
   }
 
   return (
-    <VerticalLayout className="auto-form" theme="spacing">
+    <VerticalLayout className={`flex flex-col p-m gap-m ${className}`} id={id} style={style} theme="spacing">
```

Comment:
Please add a test to verify that the props are propagated to the rendered element.